tests/test_transport_network.py

```diff
@@ -94,11 +93,6 @@ def test_cache_directory(self, transport_network_files_tuple):
 
         del transport_network
 
-        # The following fails on Windows, as it does not always
-        # release file handles properly
-        if os.name != "nt":
-            assert not cache_dir.exists()  # destructor deleted cache directory
-
```

Comment:
the `cache_dir` is not necessarily deleted at this point of code execution, any more